apps.json

```
@@ -1292,7 +1292,7 @@
         "category": "communication",
         "level": 0,
         "revision": "HEAD",
-        "state": "working",
+        "state": "notworking",
```

You really want to give up the maintenance ? If yes:

```suggestion
        "state": "notworking",
        "maintained": false,
```